client/components/banner/index.jsx

```diff
@@ -65,14 +66,16 @@ class Banner extends Component {
 	};
 
 	getHref() {
-		const { href, feature, siteSlug } = this.props;
+		const { feature, plan, siteSlug } = this.props;
+
+		const href = addQueryArgs(
```

Comment:
I see a couple of issues here.

In the previous version, we altered the URL only if `href` is not specified. This one will change it in every case, and it might lead to unexpected results. Let's make sure that if we have `href`, we return it untouched; and build  the href manually only if we haven't specifically provided it.

Also: what happens if we don't have `siteSlug`? The previous version was taking that into consideration.